Source/WebKit/WebProcess/WebPage/Cocoa/TextAnimationController.mm

```diff
@@ -112,17 +114,19 @@
     return std::nullopt;
 }
 
-void TextAnimationController::cleanUpTextAnimationsForSessionID(const WTF::UUID& sessionUUID)
+void TextAnimationController::removeTransparentMarkersForSessionID(const WTF::UUID& sessionUUID)
 {
+    if (auto iterator = m_initialAnimations.find(sessionUUID); iterator != m_initialAnimations.end())
+        removeTransparentMarkersForTextAnimationID(iterator->value);
+
     auto animationStates = m_activeTextAnimations.take(sessionUUID);
     if (animationStates.isEmpty())
         return;
     for (auto animationState : animationStates)
         removeTransparentMarkersForTextAnimationID(animationState.styleID);
 
-    auto unstyledRange = m_unstyledRanges.find(sessionUUID);
-    if (unstyledRange->value)
-        removeTransparentMarkersForTextAnimationID(unstyledRange->value->styleID);
+    if (auto iterator = m_unstyledRanges.find(sessionUUID); iterator != m_unstyledRanges.end())
+        removeTransparentMarkersForTextAnimationID(iterator->value->styleID);
```

Comment:
if `value` is `nullptr` here, it can cause a crash, right?

Reply:
How would it be null? We know we have one.